orchestrator/careplancontributor/applaunch/zorgplatform/service.go

```diff
@@ -176,16 +176,30 @@ func (s *Service) RegisterHandlers(mux *http.ServeMux) {
 
 func (s *Service) EhrFhirProxy() coolfhir.HttpProxy {
 	targetFhirBaseUrl, _ := url.Parse(s.config.ApiUrl)
-
 	proxyBasePath := "/cpc/fhir"
-
 	rewriteUrl, _ := url.Parse(s.baseURL + proxyBasePath)
-
-	return coolfhir.NewProxy("App->EHR (ZPF)", log.Logger, targetFhirBaseUrl, proxyBasePath, rewriteUrl, &stsAccessTokenRoundTripper{
+	result := coolfhir.NewProxy("App->EHR (ZPF)", log.Logger, targetFhirBaseUrl, proxyBasePath, rewriteUrl, &stsAccessTokenRoundTripper{
 		transport:          s.zorgplatformHttpClient.Transport,
 		cpsFhirClient:      s.cpsFhirClient,
 		secureTokenService: s.secureTokenService,
 	}, true)
+	// Zorgplatform's FHIR API only allows GET-based FHIR searches, while ORCA only allows POST-based FHIR searches.
```

Comment:
Is this just a temporary measure until the Zorgplatform API is updated? Otherwise we are putting patient info in the URL